lib/WALASupport/SILWalaInstructionVisitor.cpp

```diff
@@ -738,6 +738,19 @@ jobject SILWalaInstructionVisitor::visitCopyAddrInst(CopyAddrInst *CAI) {
   return Node;
 }
 
+jobject SILWalaInstructionVisitor::visitBeginUnpairedAccessInst(BeginUnpairedAccessInst *BUI) {
+
+  jobject SourceVar = findAndRemoveCAstNode(BUI->getSource().getOpaqueValue());
+
+  if (Print) {
+    llvm::outs() << "\t\t [OPERAND]:" << BUI->getSource().getOpaqueValue() << "\n";
+  }
+
+  NodeMap.insert(std::make_pair(BUI->getBuffer().getOpaqueValue(), SourceVar));
+
+  return nullptr;
```

Comment:
shouldn't this return `SourceVar` instead of `nullptr`?

Reply:
Im not sure but I think the instruction is similar to the visitBeginAccessInst

Reply:
I see. I wonder what would be the most appropriate thing to do here. I’d like to avoid returning nullptrs as much as possible, especially if there’s a meaningful object that the method can return.